package.json

```diff
@@ -35,6 +35,7 @@
     "react-hot-toast": "^2.3.0",
```

Comment:
Could you use react-hot-toast

Reply:
yup will make necessary changes

Reply:
Should we show the toast in center top or right top any Suggestion ?

Reply:
@leewan-09 updated have a look

Reply:
center top ig